src/lib/doc.ts

```diff
@@ -40,6 +40,20 @@ export const findAbstractsNode = (doc: ManuscriptNode) => {
   return findChildrenByType(doc, schema.nodes.abstracts)[0]!
 }
 
+export const findBody = (doc: ManuscriptNode) => {
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
we can turn off that rule in https://github.com/Atypon-OpenSource/manuscripts-body-editor/blob/master/.eslintrc.js#L19

Reply:
I'll file a ticket to review this.